### PR TITLE
issues templates to gather all the required information in bug reports

### PR DESCRIPTION
**Pull request by sandeepsukhani**
**What this PR does**:
Adds issues templates to gather all the required info in bug reports which otherwise requires going back and forth with users to get it.

This is how it would look like: https://github.com/sandeepsukhani/test-issues-template/issues/new/choose

### CODE REVIEW

File: .github/ ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```
Comment (sandeepsukhani) [inline .github/ ISSUE_TEMPLATE/config.yml:1]: This is to hide the highlighted link in the screenshot. Users can still create an issue with a different format by clearing out all the text from the one of the existing templates.
image

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```
Comment (sandeepsukhani) [inline .github/ISSUE_TEMPLATE/config.yml:1]: This is to hide the highlighted link in the screenshot. Users can still create an issue with a different format by clearing out all the text from the one of the existing templates.
image

Comment (pstibrany) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Let's keep this enabled. It doesn't do any harm and is sometimes useful.

**Review by pstibrany — COMMENTED** [review]
I'm not a big fan of these templates, and end up deleting the text more often than not. I'd only keep one for bug reports.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (sandeepsukhani) [reply .github/ISSUE_TEMPLATE/config.yml:1]: Sounds good!

**Comment by sandeepsukhani** [comment]
> I'm not a big fan of these templates, and end up deleting the text more often than not. I'd only keep one for bug reports.

It helps people who are new or not used to reporting bugs/requesting features to have a template for it which helps capture most of the required info. I was investigating a bug yesterday which didn't have much info so I had to go back and forth with him to get more information. For power users, it is always possible to just use a blank issue or clear out the template.

**Review by pstibrany — APPROVED** [review]

**Review by pracucci — APPROVED** [review]
Thanks!

**Pull request merged**